web/BEM-style.md

```diff
@@ -17,11 +17,11 @@ About 30 - 45 minutes
   
 ### Motivation
 
-On smaller pet projects, the way you organise your CSS or even SaSS styles doesn't really matter and it's mainly because you never really look back into the project ever again thinking about maybe scaling it or adding new features every time.
+On smaller pet projects, the way you organise your CSS or even Sass styles doesn't really matter and it's mainly because you never really look back into the project ever again thinking about maybe scaling it or adding new features every time.
```

Comment:
Could you please reword this entire section?  It's very hard to read.  Almost all sentences are run-on.

Reply:
Feel free to say it in a totally different way.